nuve/nuveAPI/resource/usersResource.js

```diff
@@ -27,19 +27,19 @@ exports.getList = function (req, res) {
     doInit(req, function (currentService, currentRoom) {
 
         if (currentService === undefined) {
-            res.send('Service not found', 404);
+            res.status(404).send('Service not found');
             return;
         } else if (currentRoom === undefined) {
             log.info('message: getUserList - room not found, roomId: ' + req.params.room);
-            res.send('Room does not exist', 404);
+            res.status(404).send('Room does not exist');
             return;
         }
 
         log.info('message: getUsersList success, roomId: ' + currentRoom._id +
                  ', serviceId: ' + currentService._id);
         cloudHandler.getUsersInRoom(currentRoom._id, function (users) {
-            if (users === 'error') {
```

Comment:
The response in "users" can only be the list of users (an array) or the string "timeout" in case there is a problem in the call to EC. 

So I think at this point it is enough with handling this kind of error